tests/acceptance/pageObjects/FilesPageElement/sharingDialog.js

```diff
@@ -324,15 +324,20 @@ module.exports = {
      * @param {string} collaborator
      */
     getDisplayedPermission: async function (collaborator) {
-      await collaboratorDialog.expandShareRoleDropdown(collaborator)
-      await this.selectRole('Custom permissions')
-      // read the permissions from the checkboxes
-      const currentSharePermissions = await this.getSharePermissions()
-
-      // Hide role select dropdown
-      this.moveToElement('@customPermissionsDrop', -9, 0)
-      this.api.mouseButtonClick()
-      this.waitForElementNotPresent('@customPermissionsDrop', 1000)
+      let currentSharePermissions = {}
+      try {
+        await collaboratorDialog.expandShareRoleDropdown(collaborator)
+        await this.selectRole('Custom permissions')
+        // read the permissions from the checkboxes
+        currentSharePermissions = await this.getSharePermissions()
+
+        // Hide role select dropdown
+        await this.moveToElement('@customPermissionsDrop', -9, 0)
+        await this.api.mouseButtonClick()
+        await this.waitForElementNotPresent('@customPermissionsDrop', 1000)
+      } catch (e) {
+        console.info('Collaborator Role is not editable!')
+      }
```

Comment:
Since there is this https://github.com/owncloud/web/issues/7063 issue on the web, I am less confident about this change.

And we don't have a test to check that after resharing with read-only permission, the collaborator role is listed as plain text. Also, I am not sure whether the role should be listed as plain text or a drop-down after resharing with read-only permission.

Reply:
CC @kulmann @pascalwengerter @phil-davis

Reply:
It would be nice if we can work out why this different behavior happens! 

@saw-jan do you see the problem manually? (Is it possible to observe both behaviors when slowly using the web UI manually?) (I want to rule out the chance that there is some transient display of UI elements which the automated test sometimes "sees" but for real users maybe the correct UI elements end up being displayed after a "flicker" of the incorrect UI elements)

Reply:
Yes, the issue can be reproduced manually. Steps to reproduce are listed here https://github.com/owncloud/web/issues/7063

Reply:
I actually don't like to have this kind of workarounds in the test.
We had a similar discussion for the e2e tests in https://confluence.owncloud.com/pages/viewpage.action?pageId=28807613

what to do with bugs discovered by e2e tests? 
- create an issue
- if possible run the journey without the step that fails
- if not then don't merge the test-PR that demonstrate the bug till the bug is fixed

We might want to make an exception here, because its not a failing test but a flaky one, but then we have to make super-sure that the workaround will be taken out  when the bug is fixed

@kulmann what do you think?